packages/travix-ui-kit/components/tooltip/tooltip.scss

```diff
@@ -127,25 +136,49 @@
 }
 
 .ui-tooltip__close-button-section {
-  display: flex;
-  flex-direction: row-reverse;
-  margin-bottom: 5px;
+  position: absolute;
+  top: 8px;
+  right: 7px;
+}
+
+.ui-tooltip__close-button {
+
 }
 
 .ui-tooltip__close-button {
   background: transparent;
   border: 0;
   cursor: pointer;
+  height: var(--tx-tooltip-close-button-size);
   outline: 0;
   padding: 0;
-}
-
-.ui-tooltip__close-button::before {
-  color: var(--tx-tooltip-close-button-color);
-  content: '✖';
-  font-size: var(--tx-tooltip-font-size);;
-}
-
-.ui-tooltip__close-button:hover::before {
-  color: var(--tx-tooltip-close-button-hover-color);
+  width: var(--tx-tooltip-close-button-size);
+
+  &::before,
+  &::after {
+    background: var(--tx-tooltip-close-button-color);
+    content: '';
+    cursor: pointer;
+    display: block;
+    height: var(--tx-tooltip-close-button-size);
+    position: absolute;
+    right: 50%;
+    top: 0;
+    width: var(--tx-tooltip-close-button-weight);
+  }
```

Comment:
Nice work 🙂

Do you know what's the impact of this change on RWD though? I think there are some CSS overrides there that can affect an UI Kit upgrade.